frontend/src/components/SponsorModal.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <v-dialog
+        v-model="show"
+        max-width="500"
+        >
+      <v-card>
+        <v-card-title class="headline">{{title}}</v-card-title>
+        <v-card-text>
+            {{body}}
+        </v-card-text>
+      </v-card>
+    </v-dialog>
+</template>
+
+<script>
+  export default {
+    props: {
+      title: String,
+      body: String,
+      value: Boolean
```

Comment:
It's not clear to see what this `value` prop is doing, as it isn't passed on from sponsor. It took me a bit of poking around to figure it out myself 😅 Maybe add a comment or something, it's not a big deal though.

Reply:
Yeah your right, ill add a comment it is really ambiguous